From 5181ea9b951b49b243d75b518ecca30e438d7778 Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Mon, 30 Mar 2020 21:58:44 +0800
Subject: [PATCH] ARM: dts: rv1126: It is better to define speed clock with
 CLK_GMAC_TX_RX

The advantage of this is that there is no need to change the dtsi,
when changing the rgmii and rmii modes, and can go to the parent
of assign CLK_GMAC_TX_RX externally.

Change-Id: I58fa5fa3b0a59be10105e0825f6e6c6b3ebbd7d0
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 arch/arm/boot/dts/rv1126.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index a291e2c57a52..11eda3d273ab 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -1533,7 +1533,7 @@
 		clocks = <&cru CLK_GMAC_SRC>, <&cru CLK_GMAC_TX_RX>,
 			 <&cru CLK_GMAC_TX_RX>, <&cru CLK_GMAC_REF>,
 			 <&cru ACLK_GMAC>, <&cru PCLK_GMAC>,
-			 <&cru RGMII_MODE_CLK>, <&cru CLK_GMAC_PTPREF>;
+			 <&cru CLK_GMAC_TX_RX>, <&cru CLK_GMAC_PTPREF>;
 		clock-names = "stmmaceth", "mac_clk_rx",
 			      "mac_clk_tx", "clk_mac_refout",
 			      "aclk_mac", "pclk_mac",
-- 
2.35.3

